javascript/ql/src/semmle/javascript/dataflow/Configuration.qll

```diff
@@ -795,7 +795,7 @@ private predicate summarizedHigherOrderCall(
 ) {
   exists(
     Function f, DataFlow::InvokeNode outer, DataFlow::InvokeNode inner, int j,
-    DataFlow::Node innerArg, DataFlow::ParameterNode cbParm, PathSummary oldSummary
+    DataFlow::Node innerArg, DataFlow::SourceNode cbParm, PathSummary oldSummary
```

Comment:
Could you explain this change? It seems unlikely that `this` would ever be used as a callback.

Reply:
True, it's unlikely. I went through and changed all places where the final argument of `argumentPassing` was implicitly cast to a `ParameterNode` and generalised it to `SourceNode`. If you like I can revert this particular change.

Reply:
I see. Probably doesn't matter, so no need to revert if it's working as it is.